### DISCUSSION TITLE
离线环境下，搭建的配置界面项目，SHIFT键+打开配置好像打不开页面

### DISCUSSION THREAD

**Discussion by LCC456** [shuax/MouseInc.Settings#101] [Q&A] [answered]
请问是否定要联网？

**Comment by LCC456** [comment]
确实是，我看有许多请求css和js文件用的是网络cdn资源。

**Comment by shuax** [comment]
https://github.com/shuax/MouseInc.Settings/blob/master/vue.config.js#L88

纯离线可能要改下这附近

**Reply by LCC456** [reply]
对，还有一个 hm.js 文件也要注释掉，在 public/index.html 文件里。

**Reply by iverjun** [reply]
纯小白,具体怎么搭建离线,感谢

**Reply by LCC456** [reply]
[MouseInc 手册](https://docs.shuax.com/MouseInc/#/#%E5%BC%80%E5%A7%8B)
![Snipaste_2021-09-30_08-58-36](https://user-images.githubusercontent.com/32818020/135368712-5d00d828-da14-437f-9c64-1358ba23252c.png)

[MouseInc 设置界面](https://github.com/shuax/MouseInc.Settings)
![1633480518(1)](https://user-images.githubusercontent.com/32818020/136122586-88e1a2ff-c00f-4f96-b309-5453ffe542fc.jpg)

**Comment by shuax** [comment, marked as answer]
加了cdn导致的无网络打不开，研究这个 https://github.com/shuax/MouseInc.Settings/commit/ca8250050d693e485d2d780bc34debedb71c94a2
